yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -227,41 +227,60 @@
             </div>
         {{end}}
 
+        <h2>Unsupported Query Constructs</h2>
         {{ if .UnsupportedQueryConstructs}}
-            <h2>Unsupported  Query Constructs</h2>
-            <p>Source database queries not supported in YugabyteDB, identified by scanning system tables.:</p>
+            <p>Source database queries not supported in YugabyteDB, identified by scanning system tables:</p>
             <table>
                 <tr>
                     <th>Construct Type</th>
                     <th>Queries</th>
+                    <th>Documentation</th>
```

Comment:
should we keep this as `Details` just to be consistent with other sections ?

Reply:
done